### PR TITLE
CDAP-5329 Dataset description docs

### PR DESCRIPTION
**Pull request by gokulavasan**
JIRA : https://issues.cask.co/browse/CDAP-5329

Passes Quick Build 3 : http://builds.cask.co/browse/CDAP-DOB182-3

### CODE REVIEW

File: cdap-examples/DataCleansing/src/main/java/co/cask/cdap/examples/datacleansing/DataCleansing.java [hunk 1]
```diff
@@ -53,8 +53,9 @@ public void configure() {
     createDataset(RAW_RECORDS, PartitionedFileSet.class, PartitionedFileSetProperties.builder()
       // Properties for partitioning
       .setPartitioning(Partitioning.builder().addLongField("time").build())
-      // Properties for file set
+        // Properties for file set
```
Comment (awholegunch) [inline cdap-examples/DataCleansing/src/main/java/co/cask/cdap/examples/datacleansing/DataCleansing.java]: Why insert spaces on this comment?

Comment (gokulavasan) [inline cdap-examples/DataCleansing/src/main/java/co/cask/cdap/examples/datacleansing/DataCleansing.java]: That was not intentional. You can remove it or I can do it

**Comment by gokulavasan** [comment]
@awholegunch Your changes looks great! Squash commits before merging PR. Thanks!

File: cdap-examples/DataCleansing/src/main/java/co/cask/cdap/examples/datacleansing/DataCleansing.java (hunk 1 above)
Comment (awholegunch) [inline cdap-examples/DataCleansing/src/main/java/co/cask/cdap/examples/datacleansing/DataCleansing.java]: I will fix.

**Comment by awholegunch** [comment]
Changes LGTM.

**Pull request merged**